.gitignore

```diff
@@ -1,56 +1,32 @@
-
-# Created by https://www.toptal.com/developers/gitignore/api/go,rust
-# Edit at https://www.toptal.com/developers/gitignore?templates=go,rust
-
-### Go ###
-# Binaries for programs and plugins
+# Binary suffixes
 *.exe
 *.exe~
 *.dll
 *.so
 *.dylib
 
-# Monero dir
+# Downloaded monero binaries
 /monero.tar.bz2
 /monero-*
 
-# bin
+# Generated binaries
 swapcli
 swapd
 swaprecover
 
 # Test binary, built with `go test -c`
 *.test
 
-# Output of the go coverage tool, specifically when used with LiteIDE
-*.out
+# Output of the go coverage tool
+/coverage.out
 /coverage.html
 
-# Dependency directories (remove the comment below to include it)
-# vendor/
-
-### Go Patch ###
-/vendor/
-/Godeps/
-
-### Rust ###
-# Generated by Cargo
-# will have compiled files and executables
-debug/
-target/
-
-# Remove Cargo.lock from gitignore if creating an executable, leave it for libraries
-# More information here https://doc.rust-lang.org/cargo/guide/cargo-toml-vs-cargo-lock.html
-Cargo.lock
-
-# These are backup files generated by rustfmt
-**/*.rs.bk
-
-# MSVC Windows builds of rustc generate these, which store debugging information
-*.pdb
-
-# End of https://www.toptal.com/developers/gitignore/api/go,rust
+# Logs from make targets
+/test.log
+/test-integration.log
 
-*.key
-log
-*.log
+# Apparently, swapd will generate libp2p keys with the names below into the current working
+# directory (happens during unit tests too).
+node.key
+xmrmaker.key
+xmrtaker.key
```

Comment:
@noot Is this what should be happening.  Do we need a ticket for this?  Where do other programs using libp2p store these keys?

Reply:
maybe we can put them in the temp dir instead, if they're just for tests? usually for a long-running node, they'll be in the basepath/datadir (which is `~/.atomicswap` for us)